docs/technical/new_instance.md

```diff
@@ -0,0 +1,188 @@
+# Creating a new Insights instance
+The overall idea is strong isolation across districts, which means some initial setup overhead.  Here's how you do it!  More detailed notes for various steps are below.
+
+### 1: Stand up an instance
+Goal: Visit the home page signed-out
+- Add domain
+- SSL certificate
+- New Heroku app with Node/Ruby buildpacks, grant access to team
+- Review and set environment variables (minimally)
+- Review and set PerDistrict values (minimally)
```

Comment:
Might be good to mention here that PerDistrict values are set in code, as opposed to ENV. Big difference from all the other steps in this section which are set through various web UIs.